_packaging/make-script.py

```diff
@@ -32,6 +32,9 @@ def make_gaphor_script():
         # Check for and remove two semicolons in path
         file.write("os.environ['PATH'] = os.environ['PATH'].replace(';;', ';')\n")
 
+        # Load gaphor with GTK3, the frozen gaphor-exe is segfaulting with GTK4
+        file.write("os.environ['GAPHOR_USE_GTK'] = '3'\n")
```

Comment:
What is the effect of this line? From what I read from the changes, Windowsa and Linux are fully running GTK4. Or is this only fixing the packaging step?

Reply:
For packaging, I am having PyInstaller use GTK3 for all platforms (macOS, Linux AppImage, and Windows). I am getting a segfault with GTK4, so we'll have to dive in to this more. I think it is still worth switching the default and updating our flatpak builds to use GTK4 though.

Reply:
Thanks, that was not completely clear to me when I scanned the code.

This is a really nice PR.